src/core/lib/security/credentials/external/aws_external_account_credentials.cc

```diff
@@ -30,6 +30,7 @@ namespace {
 const char* kExpectedEnvironmentId = "aws1";
 
 const char* kRegionEnvVar = "AWS_REGION";
+const char* kDefaultRegionEnvVar = "AWS_REGION";
```

Comment:
Shouldn't this say `AWS_DEFAULT_REGION`?

Reply:
yup